src/css/_onward.scss

```diff
@@ -271,3 +292,97 @@
     }
   }
 }
+
+.onward-docs {
+  padding-bottom: 120px;
+  &__contents {
+    white-space: nowrap;
+    margin-left: -20px;
+    padding-top: 11px;
+    overflow-x: scroll;
+    width: 100vw;
+    @include mq(tablet) {
+      width: auto;
+      overflow-x: visible;
+    }
+    @include mq(desktop) {
+      margin-left: 0;
+    }
+    @include mq(leftCol) {
+      margin-left: 20px;
+    }
+
+    &.col {
+      font-size: 0;
+    }
+    .docs-4-by-3-card {
+      display: inline-block;
+      vertical-align: top;
+      font-size: 18px;
+      line-height: 18px;
+      font-weight: 500;
+      color: #fff;
+      white-space: normal;
+      position: relative;
+      border-top: 1px solid #fb0;
+      background-color: #444;
+      background-position: center center;
+      background-repeat: no-repeat;
+      background-size: 100%;
+      background-size: cover;
```

Comment:
Actually I will just switch them to standard images… We don't need background-size cover, because they're always the same ratio anyway. And with standard img tags we get easier \@2x support…